SU2_CFD/src/solvers/CIncNSSolver.cpp

```diff
@@ -459,6 +464,13 @@ void CIncNSSolver::BC_Wall_Generic(const CGeometry *geometry, const CConfig *con
         LinSysRes(iPoint, nDim+1) += scalar_factor*dot_product;
       } // if streamwise_periodic
     }
+    else if (kind_boundary == HEAT_TRANSFER) {
+
+      /*--- Apply a weak boundary condition for the energy equation.
+      Compute the residual due to the prescribed temperature and transfer coefficient. ---*/
+      LinSysRes(iPoint, nDim+1) -= Transfer_Coefficient*Tdelta*Area;
+
+    }
```

Comment:
If this would be valid for the compressible solver let's make it available there too please.

Reply:
I introduced a `BC_HeatFlux_Generic` layer just as `BC_Wall_Generic` in the inc case. I did not test the results yet though, but I am somewhat confident that it will do. I'll post sth below when I have some results